### ISSUE TITLE
[Android] Build fails with minSdkVersion 16 and react-native 0.64

### ISSUE THREAD

**Issue by psdewar2** [Instabug/Instabug-React-Native#579]
I get an error using the latest version of instabug-reactnative (10.0.0) and react-native (0.64.0):
```
> Task :instabug-reactnative:processDebugAndroidTestManifest FAILED
[androidx.vectordrawable:vectordrawable-animated:1.0.0] /Users/psd/.gradle/caches/transforms-2/files-2.1/7cbda86d9eb88d3aaa9b911ac9fc6668/vectordrawable-animated-1.0.0/AndroidManifest.xml Warning:
        Package name 'androidx.vectordrawable' used in: androidx.vectordrawable:vectordrawable-animated:1.0.0, androidx.vectordrawable:vectordrawable:1.0.1.
/Users/psd/Documents/Lyrist/node_modules/instabug-reactnative/android/build/intermediates/tmp/manifest/androidTest/debug/manifestMerger370125502397774282.xml:5:5-74 Error:
        uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library [com.facebook.react:react-native:0.64.0] /Users/psd/.gradle/caches/transforms-2/files-2.1/6102d7d31d07033d665574f85d4a939a/jetified-react-native-0.64.0/AndroidManifest.xml as the library might be using APIs not available in 16
        Suggestion: use a compatible library with a minSdk of at most 16,
                or increase this project's minSdk version to at least 21,
                or use tools:overrideLibrary="com.facebook.react" to force usage (may lead to runtime failures)

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.
```

### Steps to Reproduce the Problem

`./gradlew assembleRelease assembleAndroidTest -DtestBuildType=release`

### Expected Behavior

Build finishes successfully and the app is started on the simulator.

### Actual Behavior

Build fails with the error message above.

### SDK Version
10.0.0

### React Native, iOS and Android Versions
Only happens using react-native 0.64.0

### Relevant code
Here is what my `android/build.gradle` looks like:
```
buildscript {
    ext {
        buildToolsVersion = "29.0.2"
        minSdkVersion = 21
        compileSdkVersion = 29
        targetSdkVersion = 29
        androidXAnnotation = "1.1.0"
        androidXBrowser = "1.0.0"
        ndkVersion = "21.1.6352462"
        kotlinVersion = "1.4.32"
    }
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath "com.android.tools.build:gradle:4.1.0"
        classpath "com.google.gms:google-services:4.3.3"
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects { ... }
```

**Comment by AliAbdelfattah** [comment]
@psdewar2 thank you for reporting this. We're looking into it.

**Comment by ahmed1hisham** [comment]
Hi @psdewar2 ,
Thank you for reporting this issue.
This is happening due to the latest RN update that drops support for all Android SDK versions earlier than version 21. We are now aware of the issue and are looking into it.

**Comment by psdewar2** [comment]
Thanks! In the meantime, I'm using this script that runs after executing a `yarn install`:

`"postinstall": "sed -i '' -e \"s/minSdkVersion 16/minSdkVersion 21/\" ./node_modules/instabug-reactnative/android/build.gradle && sed -i '' -e \"s/minSdkVersion 16/minSdkVersion 21/\" ./node_modules/react-native-splash-screen/android/build.gradle"`

Please let me know the ETA once it's determined.

**Comment by Brma1048** [comment]
This fixed it for me:
Find it here: https://github.com/wix/Detox/issues/2712#issuecomment-907871420
```
subprojects { 
    ext {
        compileSdk = rootProject.ext.compileSdkVersion
        minSdk = rootProject.ext.minSdkVersion
        targetSdk = rootProject.ext.targetSdkVersion
    }
    afterEvaluate { subproject ->
        if((subproject.plugins.hasPlugin('android') || subproject.plugins.hasPlugin('android-library'))) {
            android {
                compileSdkVersion rootProject.ext.compileSdkVersion
                buildToolsVersion rootProject.ext.buildToolsVersion
                defaultConfig {
                    minSdkVersion rootProject.ext.minSdkVersion
                    targetSdkVersion rootProject.ext.targetSdkVersion
                }
            }
        }
    }
}

**Comment by renanbronchart** [comment]
Is there any update of this fix?
@DavidMina96 , do you know when the PR fix will be merged?

**Comment by DavidMina96** [comment]
Hi @renanbronchart, 

The fix is planned to be included in our next release. In the meantime, we've created a snapshot for anyone facing this issue: `npm install https://github.com/Instabug/Instabug-React-Native#snapshot/10.13.0-INSD-7166`

**Comment by renanbronchart** [comment]
Thanks @DavidMina96 . And do you know how long will it take for a new release?

**Comment by DavidMina96** [comment]
@renanbronchart We have an ETA of 5 weeks for the upcoming release.

**Comment by renanbronchart** [comment]
Hello @DavidMina96 is it merged on the new release or not yet?

**Comment by DavidMina96** [comment]
@renanbronchart Yes, the fix is included in v11.0.0 🚀
